lab1/lab1.rst

```diff
@@ -0,0 +1,396 @@
+| CombinatonalOutput    | 2                     | The concatenation of  |
+|                       |                       | Out2-0 from the       |
+|                       |                       | combinational         |
+|                       |                       | circuit.              |
++-----------------------+-----------------------+-----------------------+
+
++-----------------------+-----------------------+-----------------------+
+| Signal Name           | Radix                 | Description           |
++-----------------------+-----------------------+-----------------------+
+| Input                 | 2                     | The concatenation of  |
+|                       |                       | In7-0.                |
++-----------------------+-----------------------+-----------------------+
+| ROMCombinationalOutpu | 2                     | The concatenation of  |
+| t                     |                       | Out2-0 from the ROM   |
+|                       |                       | combinational         |
+|                       |                       | circuit.              |
++-----------------------+-----------------------+-----------------------+
+
+To see if your circuit is correct, use the Python program,
+``tester.py``, included with assignment. To use it, type, in your
+command line, with all files in the same directory:
+
+``python tester.py correct.txt your.txt``
+
+where ``correct.txt`` is the file that contains the correct signals, and
+``your.txt`` is the name of the log file you have your signals in. For
+example, to test if your combinational circuit is correct, you would
+type:
+
+``python tester.py part1correct.txt part1.txt``
+
+if your log file was named ``part1.txt``.
+
+The tester was written for Python 2.7. If you want to use Python 3, you
+will need to encapsulate each print statement’s argument with
+parentheses. If you are using Windows, you may want to add Python to
+your system path to make testing easier, if you have not already.
+
+Resetting the Log Files
+=======================
+
+If your circuit has some errors the first time, in order to retest your
+file, you must perform the following steps:
+
+#. Delete the contents of your log file except for the headers, the
+   names of the signals.
+
+#. Reset your circuit by pressing Ctrl + R, or by going to Simulate
+   → Reset Simulation.
+
+#. Simulate again.
+
+#. Run ``tester.py`` again.
+
+If the first line of your log file has a line in which the last number
+is missing, you may safely delete it. Additionally, if you reset your
+circuit while you are still logging, you will notice dashes in the log
+file indicating when the circuit was reset. You may delete everything
+from those dashes up to the headers to clear your log file.
+
+Grading
+=======
+
+-  50% Implementation
+
+   -  12.5% for correct Part 1.
+
+   -  12.5% for correct Part 2.
+
+   -  25% for correct Part 3.
```

Comment:
I think this needs to be updated 50/50? Pick what you think is best.

Reply:
Did 50/50 because it's the same logic in two forms, doesn't make sense to weigh them differently.